docs/conf.py

```diff
@@ -75,7 +75,7 @@
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
+# pygments_style = "sphinx"
```

Comment:
Thanks for the update.
Why not just remove these 2 lines?

Is there any value in leaving them commented?

Reply:
That's the format of the conf.py file today -- with a bunch of unused commented out configuration variables. While I would strongly encourage doing something like https://github.com/pytest-dev/pytest/commit/05979bc4a97491c8b75f8baf95ec16f5b5516353, I didn't have much interaction with the maintainers prior to this and did not want to show up with a giant PR that changes everything. 😅

Reply:
Actually, I take that back. Lemme remove this.